Movie Challenge/Movie Challenge/Controllers/Home/Cells/SuggestionTableViewCell.swift

```diff
@@ -0,0 +1,72 @@
+//
+//  SuggestionTableViewCell.swift
+//  Movie Challenge
+//
+//  Created by Igor Gomes Arantes on 14/09/2018.
+//  Copyright © 2018 igor gomes arantes. All rights reserved.
+//
+
+import UIKit
+
+class SuggestionTableViewCell: UITableViewCell, UICollectionViewDelegate, UICollectionViewDataSource{
```

Comment:
To organize better you have to implement the Delegates and DataSources inside extensions blocks.